src/libs/OptionsListUtils.js

```diff
@@ -843,7 +843,6 @@ function getMemberInviteOptions(
     return getOptions([], personalDetails, {
         betas,
         searchInputValue: searchValue.trim(),
-        excludeDefaultRooms: true,
```

Comment:
Just removed this because I found that this param wasn't being used anywhere in `getOptions`